add new MOFs

ltalirz opened the pull request:
* add several new commercially available MOFs
 * add new column on commercial availability
 * add automatic fixer of atomic overlaps (via pymatgen)
 * validation of uniqueness

ltalirz committed:
add validation on github actions

ltalirz committed:
indicate commercial availability

add information on commercial availability from screening of
manufacturers on May 2021.

ltalirz committed:
add new MOFs

 * add several new commercially available MOFs
 * add new column on commercial availability
 * add automatic fixer of atomic overlaps (via pymatgen)

ltalirz committed:
add Zr-Fumarate, MIL-125-NH2

ltalirz committed:
add UIO-66-NH2, UIO-67

danieleongari reviewed (changes requested):
Thanks for the help!
What I did was to give a look to all the structures and see if I could spot weird stuff:

1) Miriam added non-CSD structures using some random number as label. Fine for the moment but we want to revise this labelling later. I initially did not want to add structures that were not in the CSD, but it may make sense to loose this rule not to miss some important MOF. 

2) VEJZOA and VEJZUG are the same CIF, despite they are labelled as ZIF-11 and -12

3) Shouldn't (2) trigger some validation of non-similarity? Maybe being ZIF you deactivate them since ZIF showed false positive match by the graph.

4) SUKXUS01 has atomic overlaps: again, why no triggered validation?

5) RUPRUQ_NH2 and RUBTAK_NH2: (5.1) please let's pick some other character than "_" or it may trigger some weird stuff when we create/split longer name (like the groups "CURATED-MOF_AAAAA_vx). (5.2) Maybe use numerical label like (1). (5.3) Please specify in the conventional name that it is 100% functionalized, as it may not be obvious.

6) RIDCEN (MIL-127) (6.1) needs to have water instead of terminal O (" Crystallographic analysis shows that the apical positions of the indium ions in the trimer building unit are statistically occupied by water molecules and by deprotonated and neutral imidazole molecules"). (6.2) Needs to check if the stochiometric of NO3 is correct. (6.3) I find it weird it is called "MIL", since it has been synthesized by Eddaoudi (who is not from Institut Lavoisier)

7) DIDBIE has solvent inside

ltalirz committed:
validate uniqueness of structures

 * use mofchecker to validate uniqueness of structure graph
 * remove zif-11 and zif-12 which were identical
 * remove solvent from DIDBIE

ltalirz committed:
fix SUKXUS01

It seems the symmetry of the cell re-added disorder in the linker even
after it was visually absent when looking in the structure in Mercury.

ltalirz committed:
fix RIDCEN
